content/docs/atom-guide/look-dev/materials/wrinkle-masks.md

```diff
@@ -0,0 +1,60 @@
+
+Each mask should be authored as a RGBA .tif file. The skin shader supports up to 4 wrinkle layers (normal and/or basecolor). Each color channel corresponds to the weight of one of the wrinkle layers. So for a material with only one wrinkle layer, all of the masks will be red-only (though the textures themselves will still be RGBA, with all black in the green, blue, and alpha channels). If a material uses two wrinkle layers (for example, one for vertical wrinkles and one for horizontal wrinkles), then each wrinkle mask will be red, green, or a combination of the two.
+
+## Wrinkle Mask Extension
+It is important that the masks use the _wrinklemask suffix and .tif extension, since the model builder currently looks for the _wrinklemask.tif extension to see if a mask that corresponds with the morph target exists. The suffix of _wrinklemask also ensures that the textures are processed and compressed appropriately by the asset processor using the correct settings from the _wrinklemask texture preset.
+
+## Wrinkle Mask File Properties
+The bit depth and resolution of the textures should not impact the functionality of the skin shader, so use whatever settings match the fidelity you need. We have only tested with 32 bits per channel, 4k textures. 4k is almost certainly excessive for a mask. 512x512 seems like it would be plenty, but it's at your discretion. You should be able to use 8 bits per channel while still having sufficient precision.
+
+## Mask Blending
+It is possible for multiple overlapping morph targets to be active at once. In this case, the contribution of each mask will be summed.
+
+As an example, let's say that for the left_eyebrow_raised morph target, the portion of the forehead directly above the left eyebrow is fully wrinkled using the first wrinkle layer. This means that portion of the mask will be bright red, indicating that the first wrinkle layer should be used for the normal. For the portion of the mask that covers the center of the forehead, the mask might be a darker, mid-range red that indicates that the middle of the forehead is only halfway to fully wrinkled when the left eyebrow is raised.
+
+The right_eybrow_raised mask might be a mirror or nearly-mirror image of the left mask, with the center of the forehead also halfway wrinkled. But if both the left eyebrow and right eyebrow are fully raised at the same time, the overlapping values in the center of the forehead from the two masks will be added together, resulting in the center of the forehead being fully wrinkled using the first wrinkle layer.
+
+When the values from multiple masks are combined, they will be clamped in the 0-1 range, so they will never exceed 1 for any given layer.
```

Comment:
- The descriptions help illustrate this case somewhat, but I think a more effective way to present this information is to have a supplemental image of what multiple blended morph targets and wrinkle layers looks like. Can you provide example images for left_eyebrow_raised and right_eyebrow_raised morph targets, and what they look like blended? 
- Also, beginner question or a reminder to users, how would I set this up in O3DE?

Reply:
I would love to have an example image, but I'm not an artist, and that's not something I'm capable of creating.